mmdet/models/losses/gfocal_loss.py

```diff
@@ -77,6 +77,36 @@ def distribution_focal_loss(pred, label):
     return loss
 
 
+@mmcv.jit(derivate=True, coderize=True)
+@weighted_loss
+def localization_distillation_loss(pred, soft_label, T, detach=True):
```

Comment:
This function can actually apply broader than only to localization distillation. It is a general KD loss. Therefore, please rename this function and rewrite its description for a more general purpose.